playwright/lib/testUtils.ts

```diff
@@ -1,5 +1,11 @@
+import { test } from "@playwright/test";
 import { createServer, IncomingMessage, ServerResponse } from "http";
 
+export function todo(title: string) {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  test.skip(title, () => {});
+}
```

Comment:
We don't have a `test.todo` method yet. They took [my suggestion](https://github.com/microsoft/playwright/issues/10918) tho.